src/logion/migration/1637834996469-MakeMetadataValueText.ts

```diff
@@ -0,0 +1,17 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class MakeMetadataValueText1637834996469 implements MigrationInterface {
+    name = 'MakeMetadataValueText1637834996469'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "public"."loc_metadata_item" ADD "value_text" text NOT NULL`);
```

Comment:
That one does not work on non-empty table:
> error: column "value_text" contains null values
There are 2 ways to work around this:
* either you create the column nullable and alter it to not null after the data copy (UPDATE)
* or you define a default (temporary) value. This can be achieved via an annotation:
```typescript
    @Column("text", { name: "value_text", default: "-" })
    value?: string;```
    To be tested - but this latter annotation will generate the expected 'DEFAULT' part  when adding the column.

Reply:
I applied the second suggestion.